src/Knp/Menu/Twig/MenuExtension.php

```diff
@@ -19,37 +19,25 @@ public function __construct(Helper $helper)
     public function getFunctions()
     {
         return array(
-            'knp_menu_get' => new \Twig_Function_Method($this, 'get'),
-            'knp_menu_render' => new \Twig_Function_Method($this, 'render', array('is_safe' => array('html'))),
+             new \Twig_SimpleFunction('knp_menu_get', array($this->helper, 'get')),
+             new \Twig_SimpleFunction('knp_menu_render', array($this->helper, 'render'), array('is_safe' => array('html'))),
```

Comment:
using methods on the extension allows faster Twig initialization (as long as the full initialization is not needed by another function used in the template) because the way Twig compiles the call does not requires rebuilding the full list of functions in the environment